presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -310,6 +310,7 @@ primaryExpression
     | NORMALIZE '(' valueExpression (',' normalForm)? ')'                                 #normalize
     | EXTRACT '(' identifier FROM valueExpression ')'                                     #extract
     | '(' expression ')'                                                                  #parenthesizedExpression
+    | GROUPING '(' (qualifiedName (',' qualifiedName)*)? ')'                              #groupingOperation
```

Comment:
Since you do not support grouping _expressions_ (ie not support for `grouping(a+b) ... group by a+b`), your code just works with `grouping(t.a) .. group by a`. Just change the check in `com.facebook.presto.sql.analyzer.AggregationAnalyzer.Visitor#visitGroupingOperation` (i added comment there too)